packages/jest-create-cache-key-function/src/index.ts

```diff
@@ -40,7 +40,19 @@ type NewGetCacheKeyFunction = (
 
 type GetCacheKeyFunction = OldGetCacheKeyFunction | NewGetCacheKeyFunction;
 
-function getGlobalCacheKey(files: Array<string>, values: Array<string>) {
+/**
+ * Computes the global cache key given  a collection of files and values.  This limits
+ * the output with a provided length.
+ * @param files list of files to read
+ * @param values list of values to add to the computation
+ * @param length length of the resulting key
+ * @returns {string} the global cache key
+ */
```

Comment:
I do not think this is necessary, because the function is not exported and is easy to read and understand what it does. Adding a comment like this increases the sizes of the package, but does not add any value.

```suggestion
```